### PR TITLE
unnethack 5.3.2

### PR DESCRIPTION
**Pull request by SeekingMeaning**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/unnethack.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Unnethack < Formula
   desc "Fork of Nethack"
   homepage "https://sourceforge.net/projects/unnethack/"
```
Comment (chenrui333) [inline Formula/unnethack.rb:3]: maybe we should also update the homepage to the github one?

Comment (SeekingMeaning) [reply Formula/unnethack.rb:3]: Oh yeah we should

Comment (chenrui333) [reply Formula/unnethack.rb]: Cool!

**Review by chenrui333 — APPROVED** [review]
💯

**Pull request closed without merging**